Requesting signoff for Gas Subsystem

Lmcarson20 opened the pull request.

jacksonrwoodard committed:
Create lukecarson-gas-signoff

jacksonrwoodard committed:
Update and rename lukecarson-gas-signoff to lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

JesseTNRoberts commented on Documentation/Signoffs/lukecarson-gas-signoff.md:
```diff
@@ -0,0 +1,96 @@
+# Gas Module Signoff
+
+## Subsystem Function
+The gas module plays a critical role in ensuring safety and well-being by detecting three gases that can be found in homes: propane, carbon monoxide, and ammonia. The module collects data on gas concentrations and wirelessly transmits it to the head unit through our wireless communication module. At the head unit, the data undergoes analysis to ensure compliance with OSHA standards for an eight-h ...
```
Why is this a single paragraph?

JesseTNRoberts reviewed (commented):
How do these communicate with the micro? Also, what resolution is required?

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 commented:
> How do these communicate with the micro? Also, what resolution is required?

This is addressed in the first subscript of the analysis section of the updated signoff

JesseTNRoberts commented on Documentation/Signoffs/lukecarson-gas-signoff.md:
```diff
@@ -0,0 +1,100 @@
+# Gas Module Signoff
+
+## Subsystem Function
```
Be brief and to the point regarding what this is supposed to do.

Lmcarson20 replied:
Fixed the first paragraph where it is brief and to the point.

JesseTNRoberts reviewed (commented):
You need to identify the resolution of your signal at the microcontroller. How small of a step size in ppm do you need to be able to detect? Can you?

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 commented:
To address the smallest ppm step size that the gas module needs to detect is under the first subscript-second paragraph in the analysis section

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

JesseTNRoberts reviewed (commented):
Paragraph format for analysis like the analog values is not appropriate. use in line equations with comments to go through the calculations.

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 commented:
I have updated the paragraph analysis to use in-line equations and comments above each solution.

JesseTNRoberts reviewed (commented):
I thought the analog input was 0-3.3

How much resolution do we need? (you have described the resolution we have, but I'm not clear on if this is enough because there is no appropriate constraint)

How does noise affect the readings? How much noise is acceptable?

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 commented:
As discussed in class, there will be a voltage divider to bring the voltage down to 3.3V so the data will be in range for the ADC on esp.

Added constraint 3 to show that the resolution will be sufficient.

Added constraint 11, to model the noise of the wall power and the affects it has on our system.

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

JesseTNRoberts reviewed (commented):
Ok, there's 12 mV. So, what? Is that good or bad and based on what?

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 commented:
Revision and further explanation of noise simulation

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

JesseTNRoberts reviewed (commented):
add filter to schematic.

Lmcarson20 committed:
Update lukecarson-gas-signoff.md

Lmcarson20 commented:
Added filter to schematic and updated BOM.
